pkg/api/file.go

```diff
@@ -67,6 +67,9 @@ func (s *server) fileUploadHandler(w http.ResponseWriter, r *http.Request) {
 	var fileSize uint64
 
 	ta := s.createTag(w, r)
+	if ta == nil {
```

Comment:
I would skip this check or add an internal server error response.

Reply:
same as above